external/include/protocols/webserver/http_server.h

```diff
@@ -93,6 +93,7 @@
 #define HTTP_CONF_CLIENT_STACKSIZE              8192
 #define HTTP_CONF_MIN_TLS_MEMORY                80000
 #define HTTP_CONF_SOCKET_TIMEOUT_MSEC           50000
+#define HTTP_CONF_KEEP_ALIVE_CONN_SOCKET_TIMEOUT_MSEC 300000
```

Comment:
I think HTTP_CONF_KEEP_ALIVE_MSEC is enough name, current is too long.

Reply:
yes, try to keep it short and can provide a comment .